operations/mimir-mixin/dashboards/remote-ruler-reads.libsonnet

```diff
@@ -83,43 +83,25 @@ local filename = 'mimir-remote-ruler-reads.json';
       .addPanel(
         local title = 'Requests / sec';
         $.panel(title) +
-        $.qpsPanel('cortex_query_scheduler_queue_duration_seconds_count{%s}' % $.jobMatcher($._config.job_names.ruler_query_scheduler)) +
-        $.panelDescription(title, description),
+        $.panelDescription(title, description) +
+        $.qpsPanel('cortex_query_scheduler_queue_duration_seconds_count{%s}' % $.jobMatcher($._config.job_names.query_scheduler))
```

Comment:
Shouldn't this refer to `$._config.job_names.ruler_query_scheduler`?

Reply:
(this also applies to a bunch of changes below - looks like they should all refer to `ruler_query_scheduler`, not `query_scheduler`)